mava/systems/ppo/sebulba/ff_ippo.py

```diff
@@ -88,24 +103,28 @@ def get_action_and_value(
 
     timestep = env.reset(seed=seeds)
 
-    next_dones = jax.tree_util.tree_map(
-        lambda x: jnp.repeat(x, num_agents).reshape(num_envs, -1),
-        timestep.last(),
-    )
+    next_dones = jnp.repeat(timestep.last(), num_agents).reshape(num_envs, -1)
 
     move_to_device = lambda x: jax.device_put(x, device=current_actor_device)
 
     # Loop till the desired num_updates is reached.
     while not thread_lifetime.should_stop():
         # Rollout
         traj: List = []
-        time_dict: Dict[str, List[float]] = {"single_rollout": [], "env_step_time": []}
+        time_dict: Dict[str, List[float]] = {
+            "single_rollout_time": [],
+            "env_step_time": [],
+            "getting_params_time": [],
+            "putting_rollout_time": [],
```

Comment:
Can we change these to just get and put
```suggestion
            "get_params_time": [],
            "put_rollout_time": [],
```